### PR TITLE
Throw an error before uploading if there is no scraper file in the directory

### LINKED ISSUE #4: Error if there is no scraper file in the current directory
Something like scraper.*

### PR DESCRIPTION
**Pull request by chris48s**
This one might have a merge conflict with #12
Let me know if I need to rebase

Closes #4

### CODE REVIEW

File: lib/morph-cli.rb [hunk 1]
```diff
@@ -4,8 +4,15 @@
 
 module MorphCLI
   def self.execute(directory, development, env_config)
+    all_paths = MorphCLI.all_paths(directory)
+
+    if not all_paths.find{ |file| /scraper\.[\S]+$/ =~ file }
```
Comment (henare) [inline lib/morph-cli.rb]: In Ruby [use `!` instead of `not`](https://github.com/bbatsov/ruby-style-guide#bang-not-not).

**Review by henare — DISMISSED** [review]
[Edit: Accidentally marked this review as approved, see below for updated review].

**Review by henare — CHANGES_REQUESTED** [review]
Thanks for this contribution @chris48s! :tada: It looks good except for some small improvements.

Here's what it looks like if you run `morph` before this PR:

```
henare@mana:~/Desktop rvm:(ruby-2.3.1) $ morph
Uploading and running...
Can't find scraper code. Expected to find a file called scraper.rb, scraper.php, scraper.py, scraper.pl, or scraper.js in the root directory
henare@mana:~/Desktop rvm:(ruby-2.3.1) $ 
```

And after this PR:

```
henare@mana:~/Desktop rvm:(ruby-2.3.1) $ morph
Can't find scraper to upload.
henare@mana:~/Desktop rvm:(ruby-2.3.1) $ 
```

I think the error message in the before scenario is better so I think we should improve this before merging :smile: There's also a minor style change I've suggested.

**Comment by chris48s** [comment]
Cheers. Updated as requested

**Review by henare — APPROVED** [review]

**Comment by henare** [comment]
Thanks @chris48s :star2: Released in 0.2.5 :tada:

**Pull request merged**